cards/src/main/resources/cards/custom/group3/all_hallows_end/minion_wee_squashling.json

```diff
@@ -11,7 +11,7 @@
     "targetSelection": "ANY",
     "spell": {
       "class": "HealSpell",
-      "value": 4
+      "value": 2
```

Comment:
Is this supposed to be changed?

Reply:
Nope, looks like it's because I hit"enter" in some regex search and it deleted or changed the line